### ISSUE TITLE
Content hash support

### ISSUE THREAD

**Issue by insin** [webpack-contrib/mini-css-extract-plugin#1]
Using `mini-css-extract-plugin@0.1.0`

Is it possible to support a content hash in the filename, like extract-text-webpack-plugin's `[contenthash]`?

Using `filename: '[name].[chunkhash:8].css'` the `.css` hash is also changing when only the JavaScript in the chunk has changed:

![conemu64_2018-03-04_21-35-20](https://user-images.githubusercontent.com/226692/36945132-01f2e4aa-1ff5-11e8-9b0c-20488d2e2180.png)

![conemu64_2018-03-04_21-42-07](https://user-images.githubusercontent.com/226692/36945134-0498c95e-1ff5-11e8-9d1f-c7ac101f5b36.png)

**Comment by developer-lindner** [comment]
Needs to be added! There were some issues with Cache Busting not working. 
contentHash helps there!

**Comment by evilebottnawi** [comment]
FYI. From slack:
```
sokra [3:19 PM]
I'll take care of `[contenthash]` this week. It will be added to webpack core, because it is also needed for JS files in combination with CSS or WASM files.
```

It is not necessary to implement this here :star: Just wait next release `webpack@4`

**Comment by marcusdarmstrong** [comment]
Shouldn't the chunk hash include the contents of the CSS sources, just as it does for the JS sources?

**Comment by Kovensky** [comment]
That's what the chunk hash does, yes. This is the content hash for the individual asset.

**Comment by marcusdarmstrong** [comment]
Yes; but the CSSModule defined by this plugin doesn't update the chunk hash—it just relies on the updateHash implementation in Module.

Perhaps that should be treated as a separate issue?

**Comment by sokra** [comment]
The impl has finished. CSS module test has also been added. Just waiting for webpack 4.3.0 release with contenthash support to send a PR.

**Comment by philipstanislaus** [comment]
@sokra thanks, this is awesome! Webpack 4.3.0 has just been released.

**Comment by bebraw** [comment]
This should work now.

**Comment by bebraw** [comment]
Sorry, my mistake. #59 has to land first.
